packetbeat/protos/tls/config.go

```diff
@@ -26,7 +26,7 @@ type tlsConfig struct {
 	config.ProtocolCommon  `config:",inline"`
 	SendCertificates       bool     `config:"send_certificates"`
 	IncludeRawCertificates bool     `config:"include_raw_certificates"`
-	Fingerprints           []string `config:fingerprints`
+	Fingerprints           []string `config:"fingerprints"`
 }
 
 var (
```

Comment:
Shouldn't `defaultConfig` contain `fingerprints: []string{"sha1"}`?

Reply:
I thought it was better to have `fingerprints: [sha1]` added by default to the config file, as it is now, and treat it as an empty list if removed. But that may not fit well with our configuration style, I see the rest of protocols don't have any other configuration option set other than ports.